README.md

```diff
@@ -48,8 +57,8 @@ COMMANDS:
 
 GLOBAL OPTIONS:
    --kubeconfig value                    path to the kubeconfig file to use for CLI requests (default: "~/.kube/config") [$KUBECONFIG]
-   --namespace value, -n value           namespace of the source resource (default: "default")
-   --target-namespace value, --tn value  target namespace in which resource will be copied (default: "default")
+   --namespace value, -n value           The namespace of the source resource (default: "default")
```

Comment:
I am not 100% sure that we need to prefix these two line with `The`

Reply:
ok in that case I'll revert this change.